rust/lance-table/src/format/manifest.rs

```diff
@@ -437,6 +480,51 @@ impl Manifest {
     pub fn should_use_legacy_format(&self) -> bool {
         self.data_storage_format.version == LEGACY_FORMAT_VERSION
     }
+
+    /// Get the summary information of a manifest.
+    ///
+    /// This function calculates various statistics about the manifest, including:
+    /// - total-records: Total number of records in the dataset
```

Comment:
looks like these docs are still not updated to snake case?

Reply:
PTAL and feel free to merge if no other comments